src/components/duelling-picklist/picklist-item/picklist-item.component.js

```diff
@@ -4,6 +4,7 @@ import { CSSTransition } from "react-transition-group";
 
 import {
   StyledPicklistItem,
+  StyledPicklistItemAsDiv,
```

Comment:
Same as above, you could simply pass `as="div"` to the PicklistItem and ensure that it get's passed down to the  `StyledPicklistItem`

Reply:
fixed, thanks!